stingray/lightcurve.py

```diff
@@ -180,7 +203,7 @@ def __neg__(self):
         >>> lc_new.counts
         array([100, 100, 100])
         """
-        lc_new = Lightcurve(self.time, -1*self.counts)
+        lc_new = Lightcurve(self.time, -1*self.counts, gti=self.gti)
```

Comment:
The GTI in the original light curve might have been changed by the user. Here we are maintaining the same GTI. Otherwise, creating a new light curve we will just create new, default GTIs from the start and stop times of the light curve.